easybuild/easyconfigs/i/icc/icc-2016.1.150-GCC-4.9.3.eb

```diff
@@ -0,0 +1,27 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'icc'
+version = '2016.1.150'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_%(version)s.tgz']
```

Comment:
wait, the tarball still only has `2016`, the same as the previous release?

please tell me that's not the case...